gitlab/tests/common.py

```diff
@@ -94,7 +94,7 @@
 ]
 
 METRICS_TO_TEST = [
-    "unicorn.workers",
+    "puma.workers",
```

Comment:
Should this metric be added to metadata.csv?

Reply:
It already is, we support puma metrics